tests/cypress/integration/sample_spec.js

```diff
@@ -0,0 +1,6 @@
+describe("Canonical home page", () => {
+  it("should go to the Canonical homepage and accept the cookie policy", () => {
+    cy.visit("/");
+    cy.acceptCookiePolicy();
```

Comment:
You could add an assertion in here (using the word should) to make it a real test as at the moment you're asking cypress to follow instructions but not checking what you expect to happen. e.g.

`cy.findByText(/Deliver, maintain,secure and sustain/).should("be.visible");` 

as after the modal closes you expect to be able to see this text. (Not a blocker just a suggestion as I know this is only for the set up).

Reply:
Cheers Beth! Would it makes sense to also check if the cookie-policy is visible? As we want to close it in this test.

Reply:
I suppose if you're able to click the "Accept blah blah" button then you know it must be there, this test could just be checking it closes as expected when you click that button? although I know it's only to check cypress is actually up and running and isn't a real test

Reply:
Makes sense, I will stick with your suggestion for now